WP_Auth0.php

```diff
@@ -372,6 +372,18 @@ public function render_form( $html ) {
 			return $html;
 		}
 
+		// If the user has a WP session, determine where they should end up and redirect.
+		if ( is_user_logged_in() ) {
+			$login_redirect = ! empty( $_REQUEST['redirect_to'] ) ?
```

Comment:
I think in logic usually best to avoid extra operands, you don't need the `!` if you switch the Ternary values.